plugins/woocommerce/src/Internal/Integrations/WPConsentAPI.php

```diff
@@ -31,19 +31,20 @@ class WPConsentAPI {
 	 */
 	public function register() {
 		add_action(
-			'plugins_loaded',
+			'init',
```

Comment:
Like in the WC_Helper_Updater class, is this change necessary? In the callback, it looks like all we're doing is adding callbacks to some other hooks that fire after init.

Reply:
cc: @layoutd

Reply:
I just took it for a spin and this seems to play nicely with Order Attribution and Complianz with WP Consent API.

❗️ HOWEVER, I was unable to test with block checkout because on this branch, the block checkout is not present:
![image](https://github.com/woocommerce/woocommerce/assets/228780/62ade658-ad41-4796-9dcd-1d95949a673e)


And if you swap branches and it was already present, this error shows:
![image](https://github.com/woocommerce/woocommerce/assets/228780/164a36e2-975a-4305-bb2c-e8f74da4afc5)

On `trunk` both cases work as expected (as did Classic Checkout, which doesn't appear here either). Not sure if this is reproducible on other installs, I saw it with TT4 and Storefront.

Reply:
I retested on a fresh install because it struck me as odd, thinking maybe it was a plugin conflict. But I get the same error. For what it's worth, the **Install Checkout** button gives this:
![image](https://github.com/woocommerce/woocommerce/assets/228780/6ea1c683-35c1-45a5-a8ed-9f9fa3910046)
![image](https://github.com/woocommerce/woocommerce/assets/228780/01e8897b-41f6-4f05-ac58-909a1faefc81)

And the documentation is on this page: https://fluidcheckout.com/docs/getting-started-fluid-checkout/#shortcode-vs-blocks

Reply:
Okay so I suppose one of the action hook changes causes this to not work.

If someone can tell me how those are registered I can further look into it.

> For what it's worth, the Install Checkout button gives this:

What should the screenshots tell me? 🤔

Reply:
Screenshots: I was just wondering why a third-party product was being recommended.

Looking at the changes in the PR, it's this one in `src/Packages.php` that causes this issue: https://github.com/woocommerce/woocommerce/pull/47113/files#diff-c3a93b21b4c564d9c866729f93fcaf38838c2a67ab30e5cf8a0bd4db7e269fdc

Order Attribution / WP Consent API integration seem unaffected by the PR.

Reply:
Same, I'm unable to test. However, changing the hook to run a bit early (priority 5) seems to fix it.